### PR TITLE
Output nan in the table when fitting fails

### PR DESCRIPTION
**Pull request by bb511**
This comes after #18 
The ValueError that is raised by the Simplex minimizer in problem WISH17701 panel 103 tube 3 calibration, peak 9 1 is now caught and a nan value is outputted in the table.

To test:

1. In example_runScript.py make sure
    - minimizers contains 'Simplex'
    - run_data = 'neutron'
2. Run example_runScript.py in the mantidpython terminal
3. Check the table comparison_weighted_v3.8_acc_nist_lower.html under column Simplex, last row, should display nan

Notes: 
- The table name is wrong, this is a known issue, changing it to neutron causes a crash #22 .
- This only catches RuntimeErrors and ValueErrors, if the fit algorithm returns any other kind of error, the problem is still there. Doing an except that catches everything is generally bad practice so I refrained from doing so here. In this case, other kind of error should be added as they are discovered.

### CODE REVIEW

File: fitbenchmarking/fitting_benchmarking.py [hunk 1]
```diff
@@ -168,35 +168,40 @@ def do_fitting_benchmark_one_problem(prob, minimizers, use_errors=True, count=0,
...
-                    min_sum_err_sq=sum_err_sq
-            else:
-                sum_err_sq = float("inf")
-                print(" WARNING: no output fit workspace")
-            print("   sum sq: {0}".format(sum_err_sq))
+            
+            sum_err_sq = -1
+            if not status == 'failed':
+                print("   params: {0}, errors: {1}".format(params, errors))
+                def sum_of_squares(values):
```
Comment (AnthonyLim23) [inline fitbenchmarking/fitting_benchmarking.py]: can we have this function definition outside of this function? I was a little surprised to find it in here.

File: fitbenchmarking/results_output.py [hunk 2]
```diff
@@ -99,6 +99,7 @@ def print_group_results_tables(minimizers, results_per_test, problems_obj, group
                                metric_type=FILENAME_SUFFIX_ACCURACY, file_extension=FILENAME_EXT_HTML)
 
         # print out accuracy summary table for this group of fit problems
+        # THIS IS BROKEN
```
Comment (AnthonyLim23) [inline fitbenchmarking/results_output.py]: what is broken?

Comment (bb511) [reply fitbenchmarking/results_output.py]: The accuracy summary table.

Comment (bb511) [reply fitbenchmarking/results_output.py]: The accuracy summary table.

Comment (AnthonyLim23) [reply fitbenchmarking/results_output.py]: what is it doing and what should it be doing?

Comment (bb511) [reply fitbenchmarking/results_output.py]: It should output a table with the most accurate minimser etc., instead it outputs gibberish.

**Comment by AnthonyLim23** [comment]
@Anders-Markvardsen  and @bb511 I just ran this and found that the `comparison_weighted_v3.8_acc_summary.html` (and the runtime summary) did not work. Do we have an issue for this?

**Comment by bb511** [comment]
Just checked, don't think we do. I will create one today.

**Comment by AnthonyLim23** [comment]
@bb511 is this the bit of code you commented? 

Other than that just move the function and I am happy with it

**Comment by bb511** [comment]
Yes, that would be the bit I commented.

**Pull request merged**